crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -1926,10 +1926,10 @@ pub fn integer_squareroot(n: u64) -> u64 {
     }
 
     let mut x = n;
-    let mut y = (x + 1) / 2;
+    let mut y = x.div_ceil(2);
```

Comment:
```suggestion
    let mut y = (x + 1).div_ceil(2);
```

I think the tests are failing due to the `+ 1` being removed